src/Compilers/CSharp/Portable/FlowAnalysis/AbstractRegionControlFlowPass.cs

```diff
@@ -27,22 +27,12 @@ public override BoundNode Visit(BoundNode node)
 
         // Control flow analysis does not normally scan the body of a lambda, but region analysis does.
         public override BoundNode VisitLambda(BoundLambda node)
-        {
-            return VisitLocalFunctionOrLambda(node.Body);
-        }
-
-        public override BoundNode VisitLocalFunctionStatement(BoundLocalFunctionStatement node)
-        {
-            return VisitLocalFunctionOrLambda(node.Body);
-        }
-
-        private BoundNode VisitLocalFunctionOrLambda(BoundBlock body)
         {
             var oldPending = SavePending(); // We do not support branches *into* a lambda.
             LocalState finalState = this.State;
             this.State = TopState();
             var oldPending2 = SavePending();
-            VisitAlways(body);
```

Comment:
The comment on line 8 mentions that any fix to this class should also be applied to AbstractRegionDataFlowPass. Does that apply for this change?

Reply:
These are actually completely different files now. I don't know when they were similar, but it hasn't been for a while. I'll delete both comments.